### PR TITLE
Remove SimInf and fix some R CMD check notes/warnings

### PR DESCRIPTION
**Pull request by mdenwood**
This PR removes the SimInf code and is a start to fixing the R CMD check notes/warnings.  I'm reasonably confident that the changes are not breaking anything, but it would be good to double check!

I'll continue this tomorrow and create a new PR for the remaining fixes (excluding the broken tests, which I will leave to you).

### CODE REVIEW

**Comment by mdenwood** [comment]
I have finished making changes (including vignettes), so I think these can be merged now.  All that is missing is to fix the errors in the package tests but I will leave that to you.

I guess at some point you might want to change your email address in the DESCRIPTION file, but that can wait I think.

File: DESCRIPTION [hunk 1]
```diff
@@ -16,11 +17,14 @@ Authors@R:
       person(given = "Ruby", family = "Graham", role = "ctb"),
       person(given = "Petter", family = "Hopp", role = "ctb"),
       person(given = "Thomas", family = "Rosendal", role = "ctb"))
-Description: A set of workflows facilitating the dataflow from livestock movement data to social network analysis and disease transmission models. 'movenet' also addresses common data issues such as the diversity of data formats and privacy preservation.
+Description: A set of workflows facilitating the dataflow from livestock
+  movement data to social network analysis and disease transmission models.
```
Comment (CarlijnB) [inline DESCRIPTION:21]: The description still refers to disease transmission models

File: vignettes/network-analysis.Rmd [hunk 2]
```diff
@@ -37,26 +37,35 @@ network
...
 
-```{R, eval = FALSE}
-# Create a network analysis report, with analyses of monthly network snapshots
+The movenet package provides a function to create a network analysis report,
+with analyses of monthly network snapshots:
+
+```{R eval=TRUE, echo=TRUE, message=FALSE, warning=FALSE, results='hide'}
 create_temporal_network_analysis_report(
   network,
-  output_file = paste0(tempdir(),"\\network_report.yml"),
```
Comment (CarlijnB) [inline vignettes/network-analysis.Rmd:55]: network_report.html, not .yml

File: DESCRIPTION [hunk 3]
```diff
@@ -1,6 +1,7 @@
 Package: movenet
 Title: A Toolkit Facilitating the Use of Livestock Movement Data in Veterinary Public Health
-Version: 0.2.0-1
+Version: 0.3.0-1
```
Comment (CarlijnB) [inline DESCRIPTION:3]: Should become 0.4.0-1 as I'm making movenet+leaflet 0.3.0-1
Also the date will now be a bit later

File: DESCRIPTION [hunk 4]
```diff
@@ -1,6 +1,7 @@
...
+Date: 2024-07-31
```
Comment (CarlijnB) [inline DESCRIPTION:4]:
```suggestion
Version: 0.4.0-1
Date: 2024-09-09
```

File: DESCRIPTION (hunk 1 above)
Comment (CarlijnB) [inline DESCRIPTION:21]:
```suggestion
  movement data to social network analysis. 
```

File: R/temporal_network_analysis_report.R [hunk 5]
```diff
@@ -45,18 +45,21 @@ create_temporal_network_analysis_report <- function(network, output_file,
   report_source <- system.file("reports/temporal_network_analysis.Rmd",
```
Comment (CarlijnB) [inline R/temporal_network_analysis_report.R:45]:
```suggestion
  report_source <- system.file("inst/reports/temporal_network_analysis.Rmd",
```

File: vignettes/movenet.Rmd [hunk 6]
```diff
@@ -19,8 +19,8 @@ knitr::opts_chunk$set(
 
 The goal of movenet is to simplify the use of livestock movement data in
 veterinary public health. It does this by providing workflows for
-pseudonymisation, network analysis, and integration of movement and/or
-holding data into transmission models. A first step in any of these
+pseudonymisation, network analysis, and processing of movement and/or
+holding data ready for use in transmission models. A first step in any of these
```
Comment (CarlijnB) [inline vignettes/movenet.Rmd:23]:
```suggestion
pseudonymisation and network analysis. A first step in any of these
```

File: vignettes/movenet.Rmd [hunk 7]
```diff
@@ -94,7 +94,7 @@ the movement data, and standardise them to movenet format, by:
 -   checking that the `date` column can be parsed as dates (using the date format 
     indicated in the configurations), and that the `weight` column is numeric.
 
--   converting dates to R Date format, to improve interoperability.
+-   converting dates to R Date format, to improve inter-operability.
```
Comment (CarlijnB) [inline vignettes/movenet.Rmd:97]:
```suggestion
-   converting dates to R Date format, to improve interoperability.
```

**Review by CarlijnB — APPROVED** [review]
Have made a couple of very minor changes but otherwise looks good; thank you @mdenwood !

**Pull request merged**